etc/mpv-powershell-completion.ps1

```diff
@@ -0,0 +1,287 @@
+#!/bin/pwsh
+
+# PowerShell command line completion for the mpv/mpv.net media player.
+
+# It can be installed by dot sourcing it in the PowerShell profile.
+
+#
+# This file is part of mpv.
+#
+# mpv is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# mpv is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+$Options = New-Object Collections.Generic.List[Object]
+$UpdateOptions = @('vo', 'ao', 'profile', 'hwdec', 'audio-device', 'scale', 'tscale',
+                   'error-diffusion', 'vulkan-device')
+
+Function SetOptions
+{
+    try
+    {
+        $optionContent = mpv --no-config --list-options
+    }
+    catch
+    {
+        throw
+    }
+
+    foreach ($line in $optionContent)
+    {
+        $line = $line.Trim()
+
+        if (-not $line.StartsWith("--"))
+        {
+            continue
+        }
+
+        $table = @{ value = ""; type = ""; choices = $null }
+
+        if ($line.Contains(" "))
+        {
+            $table["name"] = $line.Substring(2, $line.IndexOf(" ") - 2)
+            $value = $line.Substring($line.IndexOf(" ") + 1).Trim()
+
+            if ($value.Contains("("))
+            {
+                $value = $value.Substring(0, $value.IndexOf("(")).TrimEnd()
+            }
+
+            $table["value"] = $value
+        }
+        else
+        {
+            $table["name"] = $line.Substring(2)
+        }
+
+        if ($value.StartsWith("Choices:"))
```

Comment:
`$value` is not initialized when we don't go into `if ($line.Contains(" "))` block

Reply:
I'll fix it. There is also often double quotes where single quotes would be better. Shebang can be removed.

Problem is I'm really not so good with git, for instance I've forgotten to make a branch for the PR, what do I do now?

Reply:
Just use the `master` branch. Once this PR is merged/closed you will be able to reuse this branch like nothing happens.  `git commit --amend` your changes. Don't forget to change the commit name to `powershell-completion: add PowerShell command line completion` or similar that matches commit naming requirements of mpv.

Reply:
I wasn't able to squash the commits, then created a new pull request here:

https://github.com/mpv-player/mpv/pull/12967/files